test/indexer/test_hca_indexer.py

```diff
@@ -664,8 +664,8 @@ def test_cell_line_sample(self):
                     cell_lines_model_organ = set(one(contents['cell_lines'])['model_organ'])
                 else:
                     cell_lines_model_organ = {cl['model_organ'] for cl in contents['cell_lines']}
-                self.assertEqual(cell_lines_model_organ, {'blood (cell_line)'})
-                self.assertEqual(one(contents['cell_suspensions'])['organ'], ['blood (cell_line)'])
+                self.assertEqual(cell_lines_model_organ, {'blood (cell_line_0)', 'blood (cell_line_1)'})
+                self.assertEqual(one(contents['cell_suspensions'])['organ'], ['blood (cell_line_0)'])
```

Comment:
So `cell_line_0` is closer (graph-wise) to the cell suspension? IOW, is cell_line_1 a parent of cell_line_0? If so, I would swap the number so that _0 denotes the parent and _1 denotes the child. Or even better, be explicit as in `_parent` and `_child`.

Reply:
Yes, cell_line_0 is the child, I'll rename the cell line model organ's with `_parent` and `_child` to be more clear in the test.

donor_organism_0.json -> specimen_from_organism_0.json -> cell_line_1.json -> cell_line_0.json -> cell_suspension_0.json -> sequence_file_0.json & sequence_file_1.json